src/main/java/frc/robot/commands/finishScore.java

```diff
@@ -14,10 +14,12 @@ public finishScore(Lift lift) {
     addCommands(
         new InstantCommand(lift::openGrabber, lift),
         new WaitCommand(Cal.Lift.GRABBER_OPEN_TIME_SECONDS),
+        new InstantCommand(lift::closeGrabber, lift),
         new ConditionalCommand(
             new InstantCommand(() -> lift.setDesiredPosition(LiftPosition.POST_SCORE_HIGH), lift)
```

Comment:
Can you add a comment here, `// TODO consider whether we even need POST_SCORE_HIGH`